src/backend/linux_raw/fs/types.rs

```diff
@@ -573,7 +573,10 @@ pub struct Stat {
 ///
 /// [`statat`]: crate::fs::statat
 /// [`fstat`]: crate::fs::fstat
-#[cfg(all(target_pointer_width = "64", not(target_arch = "mips64")))]
+#[cfg(all(
+    target_pointer_width = "64",
+    not(any(target_arch = "mips64", target_arch = "loongarch64"))
```

Comment:
Similarly we most definitely don't want to force everyone to migrate to `statx` if they ever want their code to run on LoongArch systems. We should keep the generic `stat` definitions here, but add code for translating between the two ABIs: see [my Go adaptation a while ago](https://go.dev/cl/407694) (check `src/syscall/syscall_linux_loong64.go`) or glibc (check `sysdeps/unix/sysv/linux/fstatat64.c`). We have to add the code because we're the first 64-bit architecture without `fstat` at all.